### ISSUE TITLE
Setup Redshift Accounts for Integration Testing

### ISSUE THREAD

**Issue by AdamHaffar** [ConduitIO/conduit#1623]
[Redshift PR](https://github.com/conduitio-labs/conduit-connector-redshift/pull/52) requires instance setup to pass integration testing for the [redshift connector](https://github.com/conduitio-labs/conduit-connector-redshift) `(1.21)`.

**Comment by hariso** [comment]
I haven't used this myself, but it's promising (running a Redshift compatible service locally): https://docs.localstack.cloud/user-guide/aws/redshift/

**Comment by lovromazgon** [comment]
Yeah, we should definitely try to use localstack, we also use it in the [Kinesis connector](https://github.com/conduitio-labs/conduit-connector-kinesis/blob/main/test/docker-compose.yml).

**Comment by lovromazgon** [comment]
Closing as duplicate of https://github.com/conduitio-labs/conduit-connector-redshift/issues/58.
